source/includes/_payment.md

```diff
@@ -1761,13 +1949,21 @@ payment.remove_custom_field(custom_field_id,
 ```
 
 ```python
-paymentApi = killbill.api.PaymentApi()
+created_by = 'user'
+
+paymentApi = killbill.PaymentApi()
+
 payment_id = 'f33e0adc-78df-438a-b920-aaacd7f8597a'
 
 paymentApi.delete_payment_custom_fields(payment_id,
-                                        created_by,
-                                        api_key,
-                                        api_secret)
+                                        created_by)
+```
```

Comment:
I think the `customField` of the field to be deleted also needs to be specified here. Can you confirm and if so add this parameter? See [this thread.](https://killbillio.slack.com/archives/C02C15Z92PJ/p1692775418598969)